framework/views/errorHandler/callStackItem.php

```diff
@@ -9,13 +9,28 @@
 /* @var $end int */
 /* @var $args array */
 /* @var $handler \yii\web\ErrorHandler */
+$html = <<<HTML
+IDE
+<svg class="icon icon--new-window" focusable="false" aria-hidden="true" width="16" height="16">
+    <use href="#new-window"></use>
+</svg>
+HTML;
 ?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" hidden>
```

Comment:
What is this SVG for? I don't quite understand how does that work...

Reply:
I added an icon for an external link. It's just UI/UX thing.

It's "link" by id #new-window. I did this way to save a little bit of data/DOM size.
Info: https://www.w3.org/TR/SVG/linking.html

Excuse me, if I've not understood your question.